modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -103,6 +103,7 @@ Runtime Fabric on VMs / Bare Metal requires one of the following operating syste
 
 * Red Hat (RHEL) v7.4, v7.5, v7.6, v7.7, v7.8, v7.9, v8.0, v8.1, v8.2, v8.3
 * CentOS v7.4, v7.5, v7.6, v7.7, v7.8, v7.9, v8.0, v8.1, v8.2, v8.3
+* Ubuntu v18.04 (using installer version 1.1.1625094374-7058b20 or later)
```

Comment:
@sarjeet2013 do we need this installer note in parentheses or can I remove it?

Reply:
@digvijaysk19 need to the correct version

Reply:
Suggestion: can we use asterisks on Ubuntu 18.04 and add a note underneath it to include the statement.

* Ubuntu v18.04*
Note *: This is supported using Runtime Fabric Appliance version 1.1.1625094374-7058b20 or later.